CONTRIBUTING.md

```diff
@@ -67,3 +79,74 @@ If you want to take part of this please follow this process:
 * Create a commit with all changes.
 
 * Go to the next rubocop offense.
+
+# <a name="setup_backend"/>Setup an OBS backend for development
```

Comment:
You may develop also in backend parts and esp. with distributed setup. So we need some place somewhere how the backend gets setup.